conftest.py

```diff
@@ -218,7 +218,7 @@ def pytest_collection_modifyitems(session, config, items):
             for forced in config.option.force_execute:
```

Comment:
```suggestion
            for forced in config.option.force_execute:
                current_node = item
                while current_node.nodeid.startswith(forced):
                    current_node.own_markers = [m for m in current_node.own_markers if m.name not in ("skip", "skipif", "xfail")]
                    current_node = current_node.parent
```

Reply:
This code will activate other child method, which is not what user intended to do. Though, you're right that my previous solution was missing this "nested" use case. The last commit fixes that point (and I added few tests)